Antidote/CallsManager.m

```diff
@@ -441,6 +470,19 @@ - (AbstractCallViewController *)createViewControllerForCall:(OCTCall *)call
             activeVC.delegate = self;
             activeVC.dataSource = self;
             viewController = activeVC;
+
+            if (call.friendSendingVideo) {
+                activeVC.videoView = [self.manager videoFeed];
+            }
+
+            if (call.videoIsEnabled) {
+                __weak ActiveCallViewController *weakVC = activeVC;
+                [self.manager getVideoCallPreview:^(CALayer *layer) {
+                    ActiveCallViewController *strongVC = weakVC;
+                    strongVC.previewLayer = layer;
+                }];
+            }
```

Comment:
Hm, some code duplication with `updateCurrentCallInterface:` method. It would be better to avoid it